packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -797,10 +796,12 @@ mixin WidgetInspectorService {
   void registerServiceExtension({
     required String name,
     required ServiceExtensionCallback callback,
+    required RegisterServiceExtensionCallback registerExtension,
   }) {
-    _registerServiceExtensionCallback(
+    registerServiceExtension(
```

Comment:
It looks like this infinite recursion is causing OOM failures in the presubmit checks.